pydrive2/test/test_file.py

```diff
@@ -842,6 +842,39 @@ def test_Parallel_Insert_File_Passed_HTTP(self):
         files = self._parallel_uploader(self.FILE_UPLOAD_COUNT, 10)
         self._parallel_downloader(files, 10)
 
+    # Tests for Copy file.
+    # ====================
+
+    def test_CopyFile(self):
+        drive = GoogleDrive(self.ga)
+        file1 = drive.CreateFile()
+        filename = self.getTempFile("copytestfile")
+        content = "hello world!"
+        file1["title"] = filename
+        file1.SetContentString(content)
+        pydrive_retry(file1.Upload)
+        pydrive_retry(file1.FetchMetadata)
+
+        parent_id = file1["parents"][0]["id"]
+        parent_folder = drive.CreateFile({"id": parent_id})
+
+        file1.Copy(parent_folder, new_title="copytestfile_copy")
+
+        files = drive.ListFile({"q": f"'{parent_id}' in parents"}).GetList()
+
+        file2 = None
+        for f in files:
+            if f["title"] == "copytestfile_copy":
+                file2 = f
+                break
+        
+        self.assertIsNotNone(file2)
+        
+        pydrive_retry(file2.FetchContent)
+        self.assertEqual(file2.GetContentString(), content)
```

Comment:
should it be `file1.content, content` or something like this?

Reply:
`file1` if the file i copied (original)
`file2` is the copy

i know the content of `file1` because i had set it to `content`: 
```python
file1.SetContentString(content)
```

Now i need to test if the content of the copy is the same as the original:
```python
self.assertEqual(file2.GetContentString(), content)
```